includes/Model/Story.php

```diff
@@ -187,6 +196,17 @@ public function load_from_post( $_post ): bool {
 			}
 		}
 
+		/**
+		 * Poster.
+		 *
+		 * @var array{url?:string, width?: int, height?: int}|false $poster
+		 */
+		$poster = get_post_meta( $post->ID, Story_Post_Type::POSTER_META_KEY, true );
+		if ( $poster ) {
+			$this->poster_portrait      = $poster['url'];
+			$this->poster_portrait_size = [ $poster['width'], $poster['height'] ];
```

Comment:
```suggestion
			$this->poster_portrait_size = [ (int) $poster['width'], (int) $poster['height'] ];
```

So that there is no need to do any type casting later